backend/config.py

```diff
@@ -25,9 +25,12 @@
 
 # Which hardware board the software is being run on, for loading known hardware configuration. 
 # supported values:
-# pi, AML-S905X-CC-V1.0A, custom
+# pi, AML-S905X-CC-V1.0A, custom, simulation
 # Custom loads nothing. All hardware used must then be specified in base_hardware.yaml 
-hardwareBoard = environ.get("HARDWARE_BOARD", "pi")  
+# hardwareBoard = environ.get("HARDWARE_BOARD", "pi")  
+# environ.update({"W1THERMSENSOR_NO_KERNEL_MODULE": "1"})  
+
+hardwareBoard = environ.get("HARDWARE_BOARD", "simulation")  
```

Comment:
At the moment I think "pi" as the setting should be the default config value here, and the commented out lines should be removed

Reply:
sure, thanks